.idea/markdown-navigator/profiles_settings.xml

```diff
@@ -0,0 +1,3 @@
+<component name="MarkdownNavigator.ProfileManager">
```

Comment:
This file was probably added before .gitignore was modified. .idea folder should be removed from the repository.